### ISSUE TITLE
OS detection 2nd time with unknown value?

### ISSUE THREAD

**Issue by artem-sidorenko** [chef/kitchen-inspec#92]
### Description

After the upgrade of chefdk from 0.12 to chefdk 0.15 `os['family']` returns unknown at some point. I see this only within test kitchen and kitchen-inspec. Inspec itself doesn't show this behavior.

### InSpec and Platform Version

Inspec version:
```bash
$ chef gem list | grep inspec
debug_inspector (0.0.2)
inspec (0.26.0)
kitchen-inspec (0.14.0)
$ chef --version
Chef Development Kit Version: 0.15.15
chef-client version: 12.11.18
delivery version: 0.0.23 (bf89a6b776b55b89a46bbd57fcaa615c143a09a0)
berks version: 4.3.5
kitchen version: 1.10.0
```

### Replication Case

Create some cookbook and simple kitchen configuration, configure kitchen to use inspec as verifier. Create following test file and invoke kitchen verify:
```ruby
puts os['family']
```

Invoke kitchen verify:
```bash
$ kitchen verify packages-bento-debian-84
-----> Starting Kitchen (v1.10.0)
-----> Verifying <packages-bento-debian-84>...
       Detected alternative framework tests for `inspec`
       Use `chef-rkt/test/integration/packages/inspec` for testing
debian
unknown
...
```

Expected output:
```bash
$ kitchen verify packages-bento-debian-84
-----> Starting Kitchen (v1.10.0)
-----> Verifying <packages-bento-debian-84>...
       Detected alternative framework tests for `inspec`
       Use `chef-rkt/test/integration/packages/inspec` for testing
debian
...
```

**Comment by xTNTx** [comment]
@artem-sidorenko The core bug is described in issue above. It was present earlier but came up once default formatter was changed to `cli`. As temporary fix, change it back to `progress`, e.g. in your `.kitchen.yml`:
```
verifier:
  name: inspec
  format: progress
```

**Comment by artem-sidorenko** [comment]
@xTNTx many thx for the hint, it works:)

**Comment by xTNTx** [comment]
FIxed by chef/inspec#828
